src/frontend/komponenter/Behandling/Inngangsvilkår/Inngangsvilkår.tsx

```diff
@@ -20,15 +29,44 @@ const Inngangsvilkår: FC<Props> = ({ behandlingId }) => {
         axiosRequest<IInngangsvilkår, void>({
             method: 'GET',
             url: `/familie-ef-sak/api/vurdering/${behandlingId}/inngangsvilkar`,
-        })
-            .then((hentetInngangsvilkår: Ressurs<IInngangsvilkår>) => {
-                settInngangsvilkår(hentetInngangsvilkår);
-            })
-            .catch((error: AxiosError) => {
-                settInngangsvilkår(
-                    byggFeiletRessurs('Ukjent ved innhenting av inngangsvilkår', error)
-                );
-            });
+        }).then((hentetInngangsvilkår: Ressurs<IInngangsvilkår>) => {
+            settInngangsvilkår(hentetInngangsvilkår);
+        });
+    };
+
+    const oppdaterVurdering = (vurdering: IVurdering) => {
+        return axiosRequest<string, IVurdering>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/vurdering/inngangsvilkar`,
+            data: vurdering,
+        }).then((respons: Ressurs<string>) => {
+            if (
+                inngangsvilkår.status === RessursStatus.SUKSESS &&
```

Comment:
Er det TS som krever en sjekk på denne? Trodde det var gitt når man kom hit?

Reply:
Yep, dette pga bruket av `Ressurs`, og slik det er nå så kan Ressurs returnere 200 men en status som er feilet
Vi kan endre att Ressurs med 200 kun skal returnere en RessursSuksess, og att API'et ikke returnerer ressurs med 200 som ikke har status Suksess

Av denne grunnen er jeg veldig skeptisk kring Ressurs, vi trenger virkelig ikke den for 200 responser.
Vi kan godt returnere den ved feil.

Reply:
Samtidig har du muligheten for å enkelt sette ressurs i ett state og håntere alle ulike statuser/state for en ressurs.. Men burde gå å løse på en annen måte kanskje

Reply:
Si ifra hvis dere finner en bedre løsning ;)

Reply:
@henninghaakonsen jeg tenker den delen med att man kan håndtere hele Ressurs-statet på en måte er veldig fin, men liker egentlige ikke att backend skal returnere ressurs med status=SUKSESS når det er 200/201 :P så er vel kanskje mest negativ pga den delen haha :) 
Att vi må sjekke om det er SUKSESS før vi henter data er også i utgangspunkten greit, blir bare litt mye kode for de

Reply:
Ja, i andre prosjekter med lignende oppsett har jeg pleid å lage en `hentData(ressurs<T>, default: T)` som bidrar til at man alltid får en verdi. 

200/201 kan være litt uvant, men det er i tråd med nyere måter å hente data på som f.eks. graphql. Jeg mener også det er enklere å forholde seg til én oppførsel, kontra 2xx, 3xx, 4xx som potensielt kan ha gyldige ressurser.